### ISSUE TITLE
Mock-up the qualitative 🔴 🟡 🟢 assessments dashboard

### ISSUE THREAD

**Issue by jgaehring** [runrig-coop/rgo-early-warning-system#3]
For a first iteration on the dashboard, we decided to take a few steps towards presenting a visual mock-up to the users, so they can provide feedback:

- [x] Model the data we need to represent for each farm in this iteration
- [ ] Hardcode the HTML layout to illustrate what this will look like
- [ ] Take some screenshots of this to send to users

This is based on the requirements outlined in the [initial design doc](https://github.com/runrig-coop/open-design-workshops/tree/main/rgo-crop-plan-auditing#early-warning-system).

**Comment by jgaehring** [comment]
Ultimately, we only got the modelling part of this done, and decided to call it a day before taking a stab at the HTML, but it was a very useful exercise!

@Wright4TheJob, did you capture some screenshots or have a link to the model you sketched up?

**Comment by Wright4TheJob** [comment]
Yes, I added the graphic to the README in a recent commit.

**Comment by jgaehring** [comment]
I got Vue setup for us with #15, whenever we get that merged in, or we could just work on top of that PR as a feature branch until we've actually completed the milestone and only merge back into `main` then.

I added the Vue template right alongside the tabular and list markup, and even restored the original template from the Tauri/Vite starter, so everything's lumped together in there for now if you start it up on that branch (pro'ly need to run `npm i` first, fyi). While I'm still learning for myself how the linkage between the JS frontend and Rust backend works, I'm going to try to integrate as much as I can from those templates into the working model for the dashboard.

Perhaps we can even work out a way of using those structs you setup in `lib.rs`, @Wright4TheJob, rather than trying to reimpliment them purely in JS. At least, I think I'd find that helpful to learn from. We'll still want to work out managing it as client-side state, but in the long run I feel like it will cut our work out for us once it comes time to tackle #10.
